ps2/hangman.py

```diff
@@ -217,9 +219,13 @@ def show_possible_matches(my_word):
     '''
     # Removing white space
     my_word = ''.join(my_word.split())
+    counter = 0
     for word in wordlist:
         if match_with_gaps(my_word, word):
             print(word)
```

Comment:
maybe add some context to this print statement? Is this meant to be logging?

Reply:
I should edit the docstring for clarity.